test/api/controllers/healthController.test.ts

```diff
@@ -1,15 +1,20 @@
-import HealthController from '../../../api/controllers/healthController';
+import request from 'supertest';
+import app from '../../../api/server';
 import { getHealth } from '../../../api/services/healthService';
 
 jest.mock('../../../api/services/healthService');
 
-describe('HealthController', () => {
-  const healthController = new HealthController();
+beforeEach(() => {
+  const getHealthMock = getHealth as jest.Mock;
+  // doesn't matter what's returned, as long as it's not empty
+  getHealthMock.mockResolvedValue('');
```

Comment:
Is this required for a `200` response? Seems simpler to just leave the line: 
```
jest.mock('../../../api/services/healthService')
```

Reply:
Yeah it actually returns with `204` if the service response is undefined. I might change it to just check for a success response when I re-do these to just check the routes

Reply:
Yea as long as it doesn't throw errors here I think it is fine - you can just check that response is not null or that errors were not thrown. I also had thought about implementing mocks for each service but held off until we started making more tests since that would be a lot of work - but when that is done we can use those mocks instead of returning empty strings.